integration/helpers.go

```diff
@@ -195,6 +195,9 @@ func BuildCommonData(namespace string, options ...CommonDataOption) eventtypes.C
 				// Pod and Container name are defined by BusyboxPodCommand.
 				PodName:       "test-pod",
 				ContainerName: "test-pod",
+				PodLabels: map[string]string{
+					"run": "test-pod",
```

Comment:
Can you explain in the commitmsg which tests make use of that, or where it is checked?